app/hooks/useTrucoGame.ts

```diff
@@ -218,6 +237,98 @@ export function useTrucoGame() {
     }
   }, [gameState]);
 
+  const handleEnvido = useCallback(async () => {
+    // Can only offer envido in the first turn of a round and before truco
+    if (gameState.phase.type !== 'HUMAN_TURN' || 
+        gameState.playedCards.length > 0 || 
```

Comment:
I've found an issue where Envido in a valid state was not allowed. It happened where the CPU already played a card.
I think this condition is wrong. It should be in line to what we have in `canShowEnvidoButton` (in ActionButtons.tsx)